tests/configs.py

```diff
@@ -25,7 +25,11 @@ def run_virtual(cmd):
                            command=run_virtual("pip uninstall -y universum; " + pylint_cmd)),
                       dict(name="Run Jenkins plugin Java tests",
                            artifacts="universum_log_collapser/plugin/target/surefire-reports/*.xml",
-                           command=["mvn", "-B", "test"], directory="universum_log_collapser/plugin")])
+                           command=["mvn", "-B", "test"], directory="universum_log_collapser/plugin"),
+                      dict(name="Prepare Jenkins plugin Javascript tests project",
+                           command=["npm", "install"], directory="universum_log_collapser/e2e"),
+                      dict(name="Run Jenkins plugin Javascript tests",
+                           command=["npm", "test"], directory="universum_log_collapser/e2e")])
```

Comment:
Maybe you'll want to use configuration multiplication for this? Like moving 'npm' command and directory to upper level config?